guide/payments/receive.md

```diff
@@ -0,0 +1,213 @@
+   alt-text = "Screen asking the receiver to enter the amount being requested"
+   caption = "The receiver can specify the amount that they are requesting. This could be shared in payment links which would auto-fill the amount for the sender."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/receive/enter-details-from.png"
+   retina = "/assets/images/guide/payments/receive/enter-details-from@2x.png"
+   alt-text = "Screen asking the receiver to enter the name of the sender"
+   caption = "The receiver can specify the amount that they are requesting. This could be shared in payment links which would auto-fill the amount for the sender."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/receive/enter-details-purpose.png"
+   retina = "/assets/images/guide/payments/receive/enter-details-purpose@2x.png"
+   alt-text = "Screen asking the receiver to enter the purpose of the payment"
+   caption = "Sometimes the purpose can be determined by identifying who the payment is from. Be mindful of that, and consider easy ways to help the receiver add more context."
+   width = 250
+   height = 541
+%}
+</div>
+
+If your design’s goal is to get the receiver in the quickest time to share an address, you can choose to show them a new address straight away. The other properties can be left as optional fields. Consider the trade-off when optimizing this way, as there will be less useful contextual details for reports that you may want to provide in the future.
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/payments/receive/enter-details-compact.png"
+   retina = "/assets/images/guide/payments/receive/enter-details-compact@2x.png"
+   alt-text = "Screen showing QR Code for the address and optional entry for other properties"
+   caption = "All properties editable on a single screen"
+   width = 250
+   height = 541
+%}
+
+</div>
+
+<!-- 
+Update interopability page
+Since there are many Bitcoin applications for people to choose from, its likely that the sender and receiver are not using the same one and they don't have the abilitiy to exchange payment details within the application itself. 
+
+The exchange of payment details then most commonly happens outside of the wallet application. This 
+
+Since bitcoin is a open system and has many payment applications built ontop of it, there is a [standard format](https://bitcoin.design/guide/foundations/wallet-interoperability/#payment-links) of "payment links" that most bitcoin applications use to share payment details.
+-->
+
+## Choosing the type of address
+There are currently [three different versions](https://bitcoin.design/guide/glossary/#address) of bitcoin addresses. In order of oldest to newest they are; Legacy, Script, and SegWit. It is best practice for new applications to support SegWit, the latest, by default.
```

Comment:
Is there a reason why you decided to call this `Script`? It sounds generic. SegWit-wrapped? Curious on the source why it's `script`, is it because of `PWSH`?

Reply:
A good way to split these visually is:
- Legacy starts with 1
- P2SH starts with 3
- SegWit starts with BC1

Reply:
@pavlenex its from the glossary actually. i didn't know it was called script before, since most address types going forward are script based.

Reply:
moved this to the glossary under "address compatibility" subheading of the address definition